invenio_github/views/github.py

```diff
@@ -171,7 +171,7 @@ def init_user_github():
         try:
             github = GitHubAPI(user_id=current_user.id)
             github.init_account()
-            github.sync(async_hooks=False)
+            github.sync()
```

Comment:
Also here it might be confusing if users have to keep reloading the page several times to see their repositories load in for the first time.

Reply:
I have another small suggestion for the rewrite: Remove the `async_hooks=False` option in the sync function and the `asynchronous=False` options in `_sync_hooks()`. As far as I can tell, they are not being used anymore.